src/data_types.js

```diff
@@ -17,6 +17,12 @@ class DataType {
     switch (dataType) {
       case 'varchar':
       case 'char':
+      // mysql only
+      case 'binary':
+      // mysql only
+      case 'varbinary':
+      // postgres only
+      case 'bytea':
```

Comment:
just wondering, shouldn't we use js `Binary`?

Reply:
The three newly added `dataType`s should be mapped as `BINARY` type. On the other hand, the `jsType` of `BINARY`, which is currently listed in `src/drivers/abstract/attribute.js`, should be added as well, and `Buffer` should be preferred.

Reply:
besides unit tests regarding the change itself, integration tests need to be added as well. Such as defining model with `content: BINARY` or `avatar: BLOB`, or loading model attributes from schema info such as `foo BINARY`.